### ISSUE TITLE
Face of the giant panda sign

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7483]
**Preferred term label:**
Face of the giant panda sign

**Synonyms**
The midbrain panda sign 

**Definition (free text, please give PubMed ID)**
The face of the giant panda sign on T2-weighted MRI is characterized by preservation of normal signal intensity in the red nuclei and lateral portion of the pars reticulata of the substantia nigra, high signal in the tegmentum, and hypointensity of the superior colliculus. The findings are said to resemble the face of a giant panda.
Comment See Figure b of PMID:14557570

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:277900   PMID: 35169583  1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
